README.md

```diff
@@ -1,213 +1,111 @@
 <div align="center">
   <img alt="ReDoc logo" src="https://raw.githubusercontent.com/Redocly/redoc/master/docs/images/redoc-logo.png" width="400px" />
 
-  **OpenAPI/Swagger-generated API Reference Documentation**
-
-  [![Build Status](https://travis-ci.org/Redocly/redoc.svg?branch=master)](https://travis-ci.org/Redocly/redoc) [![Coverage Status](https://coveralls.io/repos/Redocly/redoc/badge.svg?branch=master&service=github)](https://coveralls.io/github/Redocly/redoc?branch=master) [![dependencies Status](https://david-dm.org/Redocly/redoc/status.svg)](https://david-dm.org/Redocly/redoc) [![devDependencies Status](https://david-dm.org/Redocly/redoc/dev-status.svg)](https://david-dm.org/Redocly/redoc#info=devDependencies) [![npm](http://img.shields.io/npm/v/redoc.svg)](https://www.npmjs.com/package/redoc) [![License](https://img.shields.io/npm/l/redoc.svg)](https://github.com/Redocly/redoc/blob/master/LICENSE)
+  <h2>Generate interactive API documentation from OpenAPI definitions</h2>
+  
+ </div>
+ 
+ ![ReDoc demo](https://raw.githubusercontent.com/Redocly/redoc/master/demo/redoc-demo.png)
+ 
+ **This README is for the `2.0` version of ReDoc (React-based). README for the `1.x` version is on the [v1.x](https://github.com/Redocly/redoc/tree/v1.x) branch.**
+ 
+ # About ReDoc
+ 
+TODO: Three-sentence introduction: what it does, who makes it, who it's for.
+ 
+[Try the live demo](http://redocly.github.io/redoc/)
```

Comment:
```suggestion
[Try the live demo](https://redocly.github.io/redoc/)
```
(not sure how that breaks/works w/ remote spec location etc., but should be pointed to https anyways)